### PR TITLE
Add classification for BPMN elements

### PR DESCRIPTION
**Pull request by MartijnBogaert**
OPH-175

⚠️ Linking existing BPMN element resources to the correct concepts, requires 34 different queries --> migrations have been left behind --> to get type functionality working in frontend, empty `<http://mu.semte.ch/graphs/antwerpen>` and `<http://mu.semte.ch/graphs/leuven>` graphs, and reupload BPMN files.

Corresponding PRs:
- bpmn-service: ~https://github.com/lblod/bpmn-service/pull/5~ https://github.com/lblod/bpmn-service/pull/6
- frontend: https://github.com/lblod/frontend-openproceshuis/pull/20

### CODE REVIEW

**Review by nbittich — COMMENTED** [review]
Having multiple files is a bit confusing to be honest. Why are we adding the uuids in a separate file? would it be possible to clean it up?

Also, could you put these migrations in a separate folder; we usually have the following 
tree structure:

migrations e.g
```
migrations/
     2024/
           2024-04-22-add-classification-for-bpmn-elements/
                      <your files live there>

**Comment by MartijnBogaert** [comment]
> Having multiple files is a bit confusing to be honest. Why are we adding the uuids in a separate file? would it be possible to clean it up?
> 
> Also, could you put these migrations in a separate folder; we usually have the following tree structure:
> 
> migrations e.g
> 
> ```
> migrations/
>      2024/
>            2024-04-22-add-classification-for-bpmn-elements/
>                       <your files live there>
> ```

@nbittich should be taken care of now

File: config/resources/domain.lisp [hunk 1]
```diff
@@ -33,14 +33,21 @@
   :on-path "files")
 
 (define-resource bpmnElement ()
-  :properties `((:name :string ,(s-prefix "bbo:name"))
-                (:classification :uri ,(s-prefix "rdf:type")))
+  :properties `((:name :string ,(s-prefix "bbo:name")))
   :has-many `((process :via ,(s-prefix "teamingAI:belongsToProcess")
                        :as "processes"))
+  :has-one `((bpmnElementType :via ,(s-prefix "org:classification")
```
Comment (nbittich) [inline config/resources/domain.lisp:39]: Now that I think about it, from the comment of Boris:
```
I would not use the property org:classification here though. This has a specific meaning - Indicates a classification for this Organization within some classification scheme. You’ll probably need to invent a new property instead (I’m not aware of an existing property which could be reused here. And fwiw, the organization ontology also mentions explicitly it’s also permissible to use sub-classes as a means to represent organizational categories - just to say that this issue about choosing sub-classes or a concept scheme returns in many places, and usually both approaches are ok.
```

Did you agree with Boris the `org:classification` approach is fine? unfortunately I already merged the PR in bpmn-service

**Review by nbittich — COMMENTED** [review]
Small comment regarding the chose predicate

File: config/resources/domain.lisp (hunk 1 above)
Comment (MartijnBogaert) [reply config/resources/domain.lisp]: You we right, that's something I overlooked. After consulting Boris again, we agreed to use `dct:type` instead --> the should be taken care of now

Comment (MartijnBogaert) [reply config/resources/domain.lisp]: New pr for bpmn service: https://github.com/lblod/bpmn-service/pull/6

**Comment by MartijnBogaert** [comment]
docker compose rules to test full functionality:

```yml
services:
  frontend:
    image: lblod/frontend-openproceshuis:feature-bpmn-type-classification
  bpmn:
    image: lblod/bpmn-service:feature-bpmn-element-classification

```

**Review by nbittich — APPROVED** [review]

**Pull request merged**